DOCKER.md

```diff
@@ -138,20 +150,40 @@ Replace `${HOME}/bundles` with the actual path to your bundles directory if need
 
 ```bash
 # Using echo to send a request to initialize a bundle
-echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"initialize_bundle","arguments":{"source":"/data/bundles/bundle.tar.gz"}}}' | ./scripts/run.sh
+echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"initialize_bundle","arguments":{"source":"/data/bundles/bundle.tar.gz"}}}' | \
+docker run -i --rm \
+  -v "$(pwd)/bundles:/data/bundles" \
+  -e SBCTL_TOKEN="$SBCTL_TOKEN" \
+  mcp-server-troubleshoot:latest
 ```
 
 ### Execute kubectl Commands
 
 ```bash
 # Using echo to send a request to execute a kubectl command
-echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"kubectl","arguments":{"command":"get pods"}}}' | ./scripts/run.sh
+echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"kubectl","arguments":{"command":"get pods"}}}' | \
+docker run -i --rm \
+  -v "$(pwd)/bundles:/data/bundles" \
+  -e SBCTL_TOKEN="$SBCTL_TOKEN" \
+  mcp-server-troubleshoot:latest
```

Comment:
This example won't work as written because initialize has to be done first. I think we can drop this example I don't see a way to make this easy to use.